web/admin/src/Admin.js

```diff
@@ -271,17 +271,26 @@ export default class Admin extends PureComponent {
     const { sessionId, users } = this.state
     this.props.fbc.database.private.adminableUsersRef().once('value', data => {
       const answersPerUser = data.val() || {}
+      const questions = this.questionsForCurrentSession()
       const answers = Object.keys(answersPerUser)
         .filter(
           id => answersPerUser[id].responses && answersPerUser[id].responses[sessionId] != null,
         )
         .filter(id => users[id])
-        .map(id => ({
-          firstName: users[id].firstName,
-          lastName: users[id].lastName,
-          email: users[id].email,
-          ...answersPerUser[id].responses[sessionId],
-        }))
+        .map(id => {
+          const formattedUserResponses = {}
+          Object.entries(answersPerUser[id].responses[sessionId]).forEach(item => {
```

Comment:
instead of referncing `item[0]` and `item[1]` you can destructure in the parameter list:

`...forEach(([key, value]) => {`... with better names for what the key and value represent.